cmd/snap/complete.go

```diff
@@ -28,9 +31,39 @@ func (s installedSnapName) Complete(match string) []flags.Completion {
 	return ret
 }
 
+func completeFromSortedFile(fn, match string) ([]flags.Completion, error) {
```

Comment:
s/fn/filename/ wouldn't hurt. This is used a single time, and improves the readability of the function definition.